docs/devGuide/projectManagement.md

```diff
@@ -362,6 +362,28 @@ Note that:
   * Some time later: "@all-contributors please add tlylt for doc"
   * Result: 'tlylt' will be updated to have both icons(code and doc)
 
+## Managing the repository
+
+### Projects
+As the number of issues increase, prioritising and triaging work may become more tricky even with labels. We may want to prioritise certain issues for particular version releases, or to work towards an overall vision of MarkBind. For this purpose, it may be useful to use [GitHub Projects](https://github.com/MarkBind/markbind/projects?query=is%3Aopen). 
```

Comment:
```suggestion
As the number of issues increase, prioritising and triaging work may become more tricky even with labels. We may want to prioritise certain issues for particular version releases, or to work towards an overall vision of MarkBind. For this purpose, it may be useful to use [GitHub Projects](https://github.com/MarkBind/markbind/projects?query=is%3Aopen).
````